src/main/java/com/google/api/generator/gapic/composer/rest/HttpJsonServiceCallableFactoryClassComposer.java

```diff
@@ -44,6 +46,18 @@ public static HttpJsonServiceCallableFactoryClassComposer instance() {
     return INSTANCE;
   }
 
+  @Override
+  protected List<AnnotationNode> createClassAnnotations(Service service, TypeStore typeStore) {
+    List<AnnotationNode> annotations = super.createClassAnnotations(service, typeStore);
+    // Always add @BetaApi annotation to the generated CallableFactory for now. It is a public class
```

Comment:
Could we please add the criteria for removing the `BetaApi` annotation? Consider also adding that to the generated comments so end users understand what to expect.